### PR TITLE
✨ Add support for light terminal backgrounds

### LINKED ISSUE #40: gimoji doesn't handle light themes
A screenshot says more than a multitude of words:

screenshot

Reproducible in [kitty](https://sw.kovidgoyal.net/kitty/) and gnome-terminal in Light theme, gnome-console has a slightly different theme so the light grey is *just* visible. Some squinting may be required.

### PR DESCRIPTION
**Pull request by whot**
Use termbg to detect a light vs dark terminal background and adjust the colors accordingly. This is merely the minimal implementation to make gimoji render nicely on white backgrounds but the infrastructure is mostly there for more complex styling.

Fixes #40 

Obligatory 4k-quality screenshot of my design skills:
Screenshot from 2023-08-11 12-09-24

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -17,6 +17,7 @@ crossterm = { version = "0.26.1", features = ["use-dev-tty"] }
 ratatui = "0.21.0"
 regex = { version = "1.8.2", default-features = false, features = ["std", "perf", "unicode-case" ] }
 arboard = { version = "3.2.0", default-features = false, features = ["wayland-data-control"] }
+termbg = "0.4.3"
```
Comment (zeenix) [inline Cargo.toml:20]: This bring in async-std and a lot of other deps. We don't even need any async. Considering that currently `cargo install` is the main method people can get gimoji (except on Fedora), this is problematic. Is there no other way to detect this?

What's the impact on a clean build time?

**Review by zeenix — COMMENTED** [review]
LGTM otherwise. Thanks for putting in the work.

File: Cargo.toml (hunk 1 above)
Comment (whot) [reply Cargo.toml]: Updated to use [terminal-light](https://github.com/Canop/terminal-light) instead which has lighter dependencies and still seems to work, at least in kitty, gnome-console (both light) and gnome-terminal (dark theme) as well as xterm (default white bg).

Build time hovers around the 20s mark on my machine with/without this commit now.

Comment (zeenix) [reply Cargo.toml]: I build and tested locally and it looks good. :+1:

**Review by zeenix — APPROVED** [review]

**Pull request merged**